cdap-standalone/src/main/java/co/cask/cdap/StandaloneMain.java

```diff
@@ -386,6 +402,7 @@ public static void main(String[] args) {
         System.err.println("Failed to start Standalone CDAP");
         e.printStackTrace(System.err);
       }
+      main.stopExternalProcesses();
```

Comment:
since we are calling `main.startup()` in try block, it will be nice to call `main.shutdown` here right ?